tests/ethpm/utils/test_manifest_validation_utils.py

```diff
@@ -7,10 +7,66 @@
     validate_manifest_against_schema,
     validate_manifest_deployments,
     validate_manifest_exists,
+    validate_raw_manifest_format,
 )
 from ethpm.validation import validate_manifest_version
 
 
+@pytest.fixture
+def all_raw_strict_manifests(package_names):
+    manifests = []
+    for name in package_names:
+        with open(str(V2_PACKAGES_DIR / name / "1.0.0.json")) as file_obj:
+            manifests.append(file_obj.read().strip("\n"))
+    return manifests
+
+
+@pytest.fixture
+def all_pretty_manifests(package_names):
+    manifests = []
+    for name in package_names:
+        with open(str(V2_PACKAGES_DIR / name / "1.0.0-pretty.json")) as file_obj:
+            manifests.append(file_obj.read())
+    return manifests
```

Comment:
Instead of having the fixture be a list of manifests, I think it would be better to parametrize the fixture which I believe also cleans up the tests below.

```python
@pytest.fixture.params(ALL_PACKAGE_NAMES)
def pretty_manifest(request):
    with open(str(... / request.param / "1.0.0-pretty.json")) as file_obj:
        return file_obj.read()
```

Then in your tests, if you use the `pretty_manifest` fixture, it will run the test once for each package name.